### PR TITLE
Add primitive install script

### PR DESCRIPTION
**Pull request by pietern**
Possible additional installation methods:
* Manually invoking `./install.sh`
* Pipe curl to sh: `curl -fsSL https://raw.githubusercontent.com/databricks/setup-cli/main/install.sh | sh`

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -0,0 +1,75 @@
+#!/bin/sh
+
+set -e
+
+VERSION="0.100.2"
```
Comment (andrewnester) [inline install.sh:5]: Is it possible to get latest available version from Github? So we would not need to change it on every release

Comment (pietern) [reply install.sh:5]: It is possible, but I don't think it's a good idea. By including the version in the script it is possible to lock to a certain version by referencing either the commit hash or a Git tag in this repository. It does add the overhead of performing the bump, but 1) we already have that through the VERSION field in `action.yml` and 2) we can automate that.

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -29,3 +29,44 @@ jobs:
...
+        os:
+          - macos-latest
+          - ubuntu-latest
+          - windows-latest
+
+    steps:
+      - run: curl -fsSL https://raw.githubusercontent.com/databricks/setup-cli/${{ github.sha }}/install.sh | sh
+        shell: bash
+
+      - run: databricks version
```
Comment (andrewnester) [inline .github/workflows/test.yml:71]: We might need to check the output of the command here to make sure we installed right CLI

**Review by andrewnester — APPROVED** [review]

File: .github/workflows/test.yml (hunk 2 above)
Comment (pietern) [reply .github/workflows/test.yml:71]: Filed https://github.com/databricks/setup-cli/issues/19 to track.

File: install.sh [hunk 3]
```diff
@@ -0,0 +1,75 @@
+#!/bin/sh
+
+set -e
```
Comment (fjakobs) [inline install.sh:3]: We should be more strict. I'd suggest `set -e -u -o pipefail`

See https://gist.github.com/mohanpedala/1e2ff5661761d3abd0385e8223e16425

File: install.sh [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+# Download release archive.
+curl -s -O "https://databricks-bricks.s3.amazonaws.com/v${VERSION}/${FILE}.zip"
+
+# Unzip release archive.
+unzip -q "${FILE}.zip"
+
+# Add databricks to path.
+chmod +x ./databricks
+cp ./databricks "$TARGET"
```
Comment (fjakobs) [inline install.sh:74]: We should cleanup the temp dir

**Pull request merged**